hg-fast-export.py

```diff
@@ -16,6 +16,8 @@
 import sys
```

Comment:
Corrected commit message:

Generate a git attribute entry for all lfs files

For git-lfs to work properly, each file needs extra information in the
gitattributes file. The added information is used by the git filters
to replace the current pointer file by the actual large file.

For now the gitattributes file will be the same for all branches as
the current branch isn't obviously available from the current context.